RELEASE.md

```diff
@@ -17,6 +17,13 @@ Please follow the established format:
 
 <!-- Add release notes for the upcoming release here -->
 
+# Release 3.8.1
+
+## Bug fixes and other changes
+
+- Temporarily disable internal CSS in exported SVGs to fix CORS error when exporting images (#337)
```

Comment:
minor comment: it might be helpful to mention that it will be reverted back pending the PNG export error is fixed for SVG-Crowbar 0.6.2?

Reply:
It's got a link to the PR, I considered it but in the interests of brevity I decided to leave that to people who want to look into the details and read the PR. Using the word 'temporarily' should be enough